# This message is the current pose of an object in the point cloud frame

int64 objNum
float64[3] trans
float64[4] quat
